src/main/java/frc/robot/subsystems/swerve/swervestatehelpers/AimAssistMath.java

```diff
@@ -67,4 +67,17 @@ public static double applyMagnitudeCompensation(Rotation2d velocityPerSecond, do
 			/ (magnitude + SwerveConstants.AIM_ASSIST_MAGNITUDE_FACTOR);
 	}
 
+	public static Translation2d getClosetPointInCircle(Translation2d robot, Translation2d centerOfCircle, double radius){
```

Comment:
u say in circle but its on circle (add reference to what happen when in circle)

Reply:
what?

Reply:
what happens if i am in shooting zone (in circle)?
i will get point further from my current pose which is not good